02.calendar/cal.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+require 'date'
+
+options = ARGV.getopts('m:', 'y:')
+
+if options['m'].nil? && options['y']
+  puts 'y オプションのみは対応していません'
+  return
+end
+
+today = Date.today
+
+month = options['m'].nil? ? today.month : options['m'].to_i
+year = options['y'].nil? ? today.year : options['y'].to_i
+
+first_date = Date.new(year, month)
+last_date = Date.new(year, month, -1)
+
+print_stack = []
```

Comment:
```suggestion
rows = []
```

出力する「行」が順に格納されると思うので、rowsみたいな名前の方がいいかもです

Reply:
[refactor: 変数名変更](https://github.com/sho037/ruby-practices/pull/2/commits/18171bfbb83f13b344aace6a9b1a08242553f655) こちらで変更を行いました！